components/classes/account.schema.json

```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/context/account",
+  "$id": "https://ns.adobe.com/xdm/class/account",
```

Comment:
You cannot change the $id otherwise you will break any existing schemas that referenced the old $id

Reply:
None of the account schemas are in use yet. All are experimental and we are the only ones changes these schemas.

Reply:
Will change